src/main/java/us/kbase/auth/AuthConfig.java

```diff
@@ -12,6 +12,7 @@
  * @author gaprice@lbl.gov
  *
  */
+@Deprecated
```

Comment:
Ah, this class is deprecated as well?

Reply:
yeah, should've done this in the first place. Everything except the "new" auth client should be deprecated

Reply:
where is that "new" auth client class?

Reply:
https://github.com/kbase/auth2_client_java/blob/main/src/main/java/us/kbase/auth/client/AuthClient.java